docs/core/project-sdk/msbuild-props.md

```diff
@@ -159,14 +276,194 @@ The `PackRelease` property is similar to the [PublishRelease](#publishrelease) p
+
+```xml
+<PropertyGroup>
+  <ApiCompatPreserveUnnecessarySuppressions>true</ApiCompatPreserveUnnecessarySuppressions>
+</PropertyGroup>
+```
+
+### ApiCompatRespectInternals
+
+The `ApiCompatRespectInternals` property specifies whether `internal` APIs should be checked for compatibility in addition to `public` APIs.
+
+```xml
+<PropertyGroup>
+  <ApiCompatRespectInternals>true</ApiCompatRespectInternals>
+</PropertyGroup>
+```
+
+### ApiCompatSuppressionFile
+
+The `ApiCompatSuppressionFile` item specifies the path to one or more suppression files to read from.
+
+```xml
+<ItemGroup>
+  <ApiCompatSuppressionFile Include="CompatibilitySuppressions.xml;CompatibilitySuppressions.WasmThreads.xml" />
+</ItemGroup>
+```
+
+### ApiCompatSuppressionOutputFile
+
+The `ApiCompatSuppressionOutputFile` property specifies the path to a suppression file to write to when `<ApiCompatGenerateSuppressionFile>` is `true`.
+
+### EnablePackageValidation
+
+The `EnablePackageValidation` property enables a series of validations on the package after the `pack` task. For more information, see [package validation](../../fundamentals/apicompat/package-validation/overview.md).
+
+```xml
+<PropertyGroup>
+  <EnablePackageValidation>true</EnablePackageValidation>
+</PropertyGroup>
+```
+
+### EnableStrictModeForBaselineValidation
+
+When set to `true`, the `EnableStrictModeForBaselineValidation` property enables strict mode for package baseline checks. The default is `false`.
+
+### EnableStrictModeForCompatibleFrameworksInPackage
+
+When set to `true`, the `EnableStrictModeForCompatibleFrameworksInPackage` property enables strict mode for assemblies that are compatible based on their target framework. The default is `false`.
+
+### EnableStrictModeForCompatibleTfms
+
+When set to `true`, the `EnableStrictModeForCompatibleTfms` property enables strict mode for contract and implementation assemblies for all compatible target frameworks. The default is `true`.
+
+### NoWarn
+
+The `NoWarn` property specifies the diagnostic IDs to suppress.
+
+```xml
+<PropertyGroup>
+  <NoWarn>$(NoWarn);PKV0001</NoWarn>
+</PropertyGroup>
+```
+
+### PackageValidationBaselineFrameworkToIgnore
+
+The `PackageValidationBaselineFrameworkToIgnore` item specifies a target framework to ignore from the baseline package. The framework string must exactly match the folder name in the baseline package.
+
+```xml
+<ItemGroup>
+  <PackageValidationBaselineFrameworkToIgnore Include="netcoreapp2.1;" />
+</ItemGroup>
+```
+
+### PackageValidationBaselineName
+
+The `PackageValidationBaselineName` property specifies the name of the baseline package to validate the current package against.
+
+### PackageValidationBaselinePath
+
+The `PackageValidationBaselinePath` property specifies the path to a baseline package to validate the current package against.
+
+### PackageValidationReferencePath
+
+The `PackageValidationReferencePath` item specifies the directory path where the reference assembly can be found.
```

Comment:
```suggestion
The `PackageValidationReferencePath` item specifies the directory path where the reference assembly can be found per TFM.
```

Reply:
This could also be a list of reference assembly files, comma separated: https://github.com/dotnet/sdk/blob/24e9922c9a9c8f9c004ddecb31634a70eeb5a3e0/src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs#L189.

FWIW this is a very advanced item, which you would only need to set if you invoke the `ValidatePackage` task yourself.